tools/codegen/gen.py

```diff
@@ -1102,16 +1098,6 @@ def make_file_manager(install_dir: str) -> FileManager:
             list(mapMaybe(compute_backend_select(target=Target.REGISTRATION), native_functions)),
     })
 
-    if options.force_schema_registration:
-        def computeSchemaRegister() -> Dict[str, object]:
-            schema_registrations = list(mapMaybe(
-                compute_type_method(None, target=Target.REGISTRATION, selector=SelectiveBuilder.get_nop_selector(), def_only=True),
-                native_functions))
-            return {
-                'schema_registrations': schema_registrations,
-            }
-        cpu_fm.write('SchemaRegister.cpp', computeSchemaRegister)
```

Comment:
The old SchemaRegister.cpp file used the `TORCH_LIBRARY_FRAGMENT_THIS_API....` macro instead of `TORCH_LIBRARY`, so merging all of the logic into TypeDefault.cpp means that we no longer use that macro.

Just checking, but I actually think that's fine in this case- right? That macro lets us add multiple blocks that use the same namespace. My guess as to why we originally used that macro in SchemaRegister.cpp is because of the fact that we explicitly had op registration blocks for `aten` in two files- and now we only have it in one.

Reply:
Yep that's right.